### PR TITLE
quo 427 noredink ui add support for skipping animation in ballon

### PR DESCRIPTION
**Pull request by ArthurJordao**
# :wrench: Modifying a component

## Context
Adding option to skip balloon animation in both Mark, and Block components.
## :framed_picture: What does this change look like?
No visual changes, just avoiding animations programatically.

## Component completion checklist

- [x] I've gone through the relevant sections of the [Development Accessibility guide](https://paper.dropbox.com/doc/Accessibility-guide-4-Development--BiIVdijSaoijjOuhz3iTCJJ1Ag-rGoHpC91pFg3zTrYpvOCQ) with the changes I made to this component in mind
- [x] Changes are clearly documented
  - [x] Component docs include a changelog
  - [x] Any new exposed functions or properties have docs
- [x] Changes extend to the Component Catalog
  - [x] The Component Catalog is updated to use the newest version, if appropriate
  - [x] The Component Catalog example version number is updated, if appropriate
  - [x] Any new customizations are available from the Component Catalog
  - [x] The component example still has:
    - an accurate preview
    - valid sample code
    - correct keyboard behavior
    - correct and comprehensive guidance around how to use the component
- [x] Changes to the component are tested/the new version of the component is tested
- [x] Component API follows standard patterns in noredink-ui
  - e.g., as a dev, I can conveniently add an `nriDescription`
  - and adding a new feature to the component will _not_ require major API changes to the component
- [x] If this is a new major version of the component, our team has stories created to upgrade all instances of the old component. Here are links to the stories:
  - [Issue](https://linear.app/noredink/issue/QUO-427/noredink-ui-add-support-for-skipping-animation-in-ballon-labels-for)
- [x] Please assign the following reviewers:
  - [x]  Someone from your team who can review your PR in full and review requirements from your team's perspective.
  - [x]  [team-accessibilibats-a11ybats](https://github.com/orgs/NoRedInk/teams/team-accessibilibats-a11ybats) - Someone from this group will review your PR for accessibility and adherence to component library foundations.
  - [x]  If there are user-facing changes, a designer. (You may want to direct your designer to the [deploy preview](https://github.com/NoRedInk/noredink-ui#reviews--preview-environments) for easy review):
    - For writing-related component changes, add Stacey (staceyadams)
    - For quiz engine-related components, add Ravi (ravi-morbia)
    - For a11y-related changes to general components, add Ben (bendansby)
    - For general component-related changes or if you’re not sure about something, add the Design group (NoRedInk/design)

### CODE REVIEW

**Comment by ArthurJordao** [comment]
@tesk9 sorry, I forgot to link the issue, but it is the same issue as the one I am currently working right now

**Comment by tesk9** [comment]
Awesome, thanks!

**Review by juanedi — COMMENTED** [review]
Changes in `Mark` look good to me! Probably good to merge once we use the real value in `Block` 🤘 

**With that said**, the omission made me suspect that we could probably add a toggle for the new flag in the catalog page for `Block`. Being able to see it in action before shipping seems much safer, plus it's nice to play with all the knobs as a user of the module.

Thanks a lot! 🙏

File: src/Nri/Ui/Mark/V6.elm [hunk 1]
```diff
@@ -0,0 +1,622 @@
+module Nri.Ui.Mark.V6 exposing
```
[the remark this answers is not in the document]
Comment (ArthurJordao) [reply src/Nri/Ui/Mark/V6.elm:1]: ohhh yeah, it is way better, sorry for that

[the remark this answers is not in the document]
Comment (juanedi) [reply src/Nri/Ui/Mark/V6.elm:1]: hahah no worries! `diff`ing locally isn't actually that hard 💛

**Review by charbelrami — APPROVED** [review]
Looks good!

**Pull request merged**